src/timepicker/timepicker.component.ts

```diff
@@ -0,0 +1,54 @@
+import {
+	Component,
+	Input,
+	Output,
+	EventEmitter,
+	HostBinding
+} from "@angular/core";
+
+@Component({
+	selector: "ibm-timepicker",
+	template: `
+			<div
+				[ngClass]="{
+					'bx--select--light': theme === 'light',
+					'bx--skeleton': skeleton
+				}"
+				class="bx--time-picker__input">
+				<label *ngIf="!skeleton" [attr.for]="id" class="bx--label">{{label}}</label>
+				<input
+					#timePicker
+					[value]="value"
+					[placeholder]="placeholder"
+					[pattern]="pattern"
+					[attr.id]="id"
+					[disabled]="disabled"
+					maxlength="5"
+					(change)="onChange($event)"
+					type="string"
```

Comment:
should be `type="time"` or `type="text"` ... `"string"` isn't a valid input type